streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -536,6 +539,7 @@ abstract class DStream[T: ClassTag] (
     logDebug(s"${this.getClass().getSimpleName}.readObject used")
     ois.defaultReadObject()
     generatedRDDs = new HashMap[Time, RDD[T]]()
+    recoveredReports = new HashMap[Time, StreamInputInfo]()
   }
```

Comment:
use recoveredReports to hold recovered report information. We can not report to `inputInfoTracker`, as  jobScheduler not yet initialized here.